src/og/control/LayerAnimation.ts

```diff
@@ -77,41 +77,41 @@ class LayerAnimation extends Control {
         }
     }
 
-    oninit() {
+    override oninit() {
         super.oninit();
         this.onactivate();
         this._initLayers();
         this._onLayerLoadend_ = this._onLayerLoadend.bind(this);
-        this.planet.events.on("layerloadend", this._onLayerLoadend_, this);
+        this.planet?.events.on("layerloadend", this._onLayerLoadend_, this);
```

Comment:
If you it's interesting, it's easy to refactor with arrow function, and get rid of native js signature)) but it's fine in this stage.